### DISCUSSION TITLE
Feature Request: allow mixed-case tab labels

### DISCUSSION THREAD

**Discussion by ASNNetworks** [kinghat/tabbed-card#38] [Feature Requests]
Hi, 

So I really dig this custom card! I used state-switch with custom button cards to create own tabs, but this card makes it a lot easier and manageable, great work! One thing I notice I hope to see changed is the labels. Everything is forced in CAPS while I want it to be case sensitive. Could you change it so the label is identical to the input? This way people can decide themselves if they want all caps, lower caps or start with a cap. 

Also I notice the label and icon are not aligned in the center. It's now aligned at the top, which makes it look kinda messy.
![image](https://user-images.githubusercontent.com/54230480/201519842-c5c74f4b-7274-40fe-adc8-17dca6c49d7b.png)

Also, the style option only affects the text-color, but not the icon. Could you also provide a variable to alter the icon color? 
Currently the text changes color, but the icon is hardcoded (the unselected tab). When I use a dark mode theme, the icon becomes invisible due to the hardcoded color.
![image](https://user-images.githubusercontent.com/54230480/201525155-e13e8b3c-1b3c-45ae-95bc-a38c9acfc920.png)

**Comment by ASNNetworks** [comment]
Edit: the icon and text alignment seems to be browser based. On Desktop it shows like the screenshot, on mobile however it's aligned properly. Hard to say what the issue might be.

**Comment by kinghat** [comment]
> Hi,
> 
> So I really dig this custom card! I used state-switch with custom button cards to create own tabs, but this card makes it a lot easier and manageable, great work! One thing I notice I hope to see changed is the labels. Everything is forced in CAPS while I want it to be case sensitive. Could you change it so the label is identical to the input? This way people can decide themselves if they want all caps, lower caps or start with a cap.

i moved your feature requests here to keep the issues clean. most of the tab strings in home assistant are uppercase. devices and services are mixed case. i may end up adding a feature for case sensitivity in the future. also, im not setting the case, its being done via the underlying component.

> Also I notice the label and icon are not aligned in the center. It's now aligned at the top, which makes it look kinda messy. ![image](https://user-images.githubusercontent.com/54230480/201519842-c5c74f4b-7274-40fe-adc8-17dca6c49d7b.png)
> 
> Also, the style option only affects the text-color, but not the icon. Could you also provide a variable to alter the icon color? Currently the text changes color, but the icon is hardcoded (the unselected tab). When I use a dark mode theme, the icon becomes invisible due to the hardcoded color. ![image](https://user-images.githubusercontent.com/54230480/201525155-e13e8b3c-1b3c-45ae-95bc-a38c9acfc920.png)


> Edit: the icon and text alignment seems to be browser based. On Desktop it shows like the screenshot, on mobile however it's aligned properly. Hard to say what the issue might be.


can you split this out from this feature request and open an issue for it please?

**Comment by cloudbr34k84** [comment]
This would be great as im loving this card as well and redesigning my app flows etc
